missilewars-plugin/src/main/java/de/butzlabben/missilewars/listener/game/LobbyListener.java

```diff
@@ -96,6 +97,14 @@ public void onRespawn(PlayerRespawnEvent event) {
         event.setRespawnLocation(getGame().getLobby().getSpawnPoint());
     }
 
+    @EventHandler
+    public void onInventoryOpen(InventoryOpenEvent event) {
+        Player player = (Player) event.getPlayer();
+        if (!isInLobbyArea(player.getLocation())) return;
+
+        if (player.getGameMode() != GameMode.CREATIVE) event.setCancelled(true);
+    }
```

Comment:
If an inventory-openning should be allowed in the lobby for the game-design, we have to differentiate that again (maybe with new lobby-options), but currently normal interactions are already prevented by the `InventoryClickEvent` cancel anyway.